CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 ## Next
 
+* Remove partner ID from user agent string
```

Comment:
Nit: We usually add a link to the PR at the end of the line (similar to below :))

Reply:
Just added it